test/navigation/NavigatorSpec.scala

```diff
@@ -34,14 +34,14 @@ class NavigatorSpec extends SpecBase {
           .onPageLoad(NormalMode)
       }
 
-      "must go from TaxableProfit page to InputScreen page" in {
-        navigator.nextPage(TaxableProfitPage, NormalMode, UserAnswers("id")) mustBe routes.InputScreenController
+      "must go from TaxableProfit page to Distribution page" in {
+        navigator.nextPage(TaxableProfitPage, NormalMode, UserAnswers("id")) mustBe routes.DistributionController
           .onPageLoad(NormalMode)
       }
 
```

Comment:
Where is the test to go from Distribution page to Distributions Included page?

Reply:
Test added

Reply:
Please add as table for all scenarios

 UserAnswers("id").set(DistributionPage, Distribution.Yes)
 UserAnswers("id").set(DistributionPage, Distribution.No)
UserAnswers("id").set(DistributionPage, "unknown")

Reply:
Done

Reply:
Done